packages/grant-explorer/src/features/attestations/utils/constants.ts

```diff
@@ -1,4 +1,4 @@
-// export const AttestationChainId = 42161;
-export const AttestationChainId = 11155111;
+export const AttestationChainId = 42161;
+// export const AttestationChainId = 11155111;
 export const AttestationFee = BigInt(0.0007722 * 10 ** 18);
-export const FeeExemptAttestationsLimit = 20;
+export const FeeExemptAttestationsLimit = 30;
```

Comment:
@nijoe1 made this 30 so that the first few test attestations won't affect the fee

Reply:
Cool let me put some other changes in here! And then we merge!